react-native-applovin-max.podspec

```diff
@@ -10,11 +11,27 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "11.0" }
   s.source       = { :git => "https://github.com/AppLovin/AppLovin-MAX-React-Native.git", :tag => "release_4_1_5" }
-  
+
   s.source_files  = "ios/AppLovinMAX*.{h,m}"
 
-  s.dependency "React"
+  s.dependency "React-Core"
   s.dependency "AppLovinSDK", "11.7.0"
+
+  # Don't install the dependencies when we run `pod install` in the old architecture.
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
```

Comment:
Are you sure we need all this? When I first created the RN project I tried to keep the project as slim as possible - only including dependencies, config, etc logic that is actually reqiured to run the project.

I don't think we'll ever need `RCT_NEW_ARCH_ENABLED` for a plugin project (no need to optimize JS <-> native bridge).